src/test/java/com/github/princesslana/eriscasper/data/resource/TestGuildMemberWithGuildId.java

```diff
@@ -0,0 +1,37 @@
+package com.github.princesslana.eriscasper.data.resource;
+
+import com.github.princesslana.eriscasper.data.DataAssert;
+import com.github.princesslana.eriscasper.data.Snowflake;
+import java.util.Optional;
+import org.testng.annotations.Test;
+
+public class TestGuildMemberWithGuildId {
+  @Test
+  public void fromJson_whenValidPayload_shouldDeserialize() {
+    String payload =
```

Comment:
I think this can be formatted better to make it clear of the json structure here.

Reply:
Right, you can put spaces inside the quotes, as has been done for the other json structures in tests